contracts/workflows/RoyaltyWorkflows.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// solhint-disable-next-line max-line-length
+import { AccessManagedUpgradeable } from "@openzeppelin/contracts-upgradeable/access/manager/AccessManagedUpgradeable.sol";
+import { ERC165Checker } from "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import { MulticallUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/MulticallUpgradeable.sol";
+import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+// solhint-disable-next-line max-line-length
+import { IGraphAwareRoyaltyPolicy } from "@storyprotocol/core/interfaces/modules/royalty/policies/IGraphAwareRoyaltyPolicy.sol";
+import { IIpRoyaltyVault } from "@storyprotocol/core/interfaces/modules/royalty/policies/IIpRoyaltyVault.sol";
+import { IRoyaltyModule } from "@storyprotocol/core/interfaces/modules/royalty/IRoyaltyModule.sol";
+
+import { Errors } from "../lib/Errors.sol";
+import { IRoyaltyWorkflows } from "../interfaces/workflows/IRoyaltyWorkflows.sol";
+
+/// @title Royalty Workflows
+/// @notice Each workflow bundles multiple core protocol operations into a single function to enable one-click
+/// IP revenue claiming in the Story Proof-of-Creativity Protocol.
+contract RoyaltyWorkflows is IRoyaltyWorkflows, MulticallUpgradeable, AccessManagedUpgradeable, UUPSUpgradeable {
+    using ERC165Checker for address;
+
+    /// @dev Storage structure for the RoyaltyWorkflows
+    /// @param nftContractBeacon The address of the NFT contract beacon.
+    /// @custom:storage-location erc7201:story-protocol-periphery.RoyaltyWorkflows
+    struct RoyaltyWorkflowsStorage {
+        address nftContractBeacon;
```

Comment:
what is the `nftContractBeacon` variable used for?

Reply:
good catch! We don’t actually need it in this contract. Will remove it :)